### DISCUSSION TITLE
Policy for localization approvers (to be confirmed by candidates)

### DISCUSSION THREAD

**Discussion by seokho-son** [cncf/glossary#723] [General]
## Policy for Localization Approvers

The policies and guidelines provided below apply to candidates for [Localization Approvers](https://glossary.cncf.io/contributor-ladder/#localization-approvers) ([project details](https://github.com/cncf/glossary/projects/2)).

### Important Notices
 - All candidates must **leave a comment** on the relevant PR (pertaining to update ownership) to acknowledge their understanding of these policies and guidelines. Please note that without this confirmation, the PR will not be merged. (Localization teams: ensure this message is conveyed to all candidates.)
 - Localization team approvers are expected to **exercise their permissions responsibly**.
   - As approvers, you will be granted push access to this repository, enabling you to manage (merge) PRs for localization contributions in your respective development branches.
   - However, merging PRs into the `main` branch is not permitted for localization approvers. Although you may review PRs targeting the `main` branch, you should refrain from approving them unless they pertain to your specific localization work.
 - Upon the merging of the PR, new approvers will receive a GitHub invitation to collaborate from the settings[bot] of the CNCF Glossary. It is essential to accept this invitation, which you can find via email or GitHub notifications. Failing to accept will prevent maintainers from designating you as an approver for the `dev-xx` branch.

---

### PR Merge Conditions for the `main` Branch
1. Only the Glossary Maintainers and English approvers are authorized to merge PRs into the `main` branch.
2. A minimum of one approval from the CODEOWNERS is required. CODEOWNERS are listed here for all files, including those containing English content: https://github.com/cncf/glossary/blob/main/CODEOWNERS#L13 (equivalent to chairs).
3. A minimum of two approving reviews from approvers is required.
4. While localization team approvers are technically able to approve PRs for any file, they should not do so unless the PR is relevant to their localization tasks.
   - Even with an approval from localization team approvers, they are not permitted to merge PRs into `main` branch, as these actions are restricted.
   - The policy regarding the merging of PRs may evolve over time, but the integrity of the `main` branch is ensured by the chairs.

### PR Merge Conditions for `dev-xx` Localization Branches (e.g., dev-ko)
1. The glossary maintainers have the authority to merge PRs across all branches.
2. At least one approving review from localization CODEOWNERS is necessary:
    - For instance, CODEOWNERS for Korean content are specified here: https://github.com/cncf/glossary/blob/main/CODEOWNERS#L54 (the same individuals serve as Korean approvers).
    - If a PR include files beyond the scope of localization, the Glossary maintainers are required to conduct a review and perform the merge.
3. A minimum of two approving reviews from localization approvers is required.
4. Localization teams have the capability to merge PRs within their development branches. However, if a PR includes files not related to localization, approvals from maintainers (who are codeowners for all files) are required.
5. Ensure there is one commit per PR when localizing a document. Use the `Squash and merge` method for merging PRs.

---

Please be aware that we employ CODEOWNERS to automatically solicit reviews from the appropriate individuals.
(For more information on CODEOWNERS, visit [GitHub's documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#about-code-owners))

Actual access permissions are managed through the repository settings of the CNCF Glossary repo.
- Permission changes can be made using the GitHub UI; however, we utilize the [Settings app](https://github.com/apps/settings).
- The [Settings app](https://github.com/apps/settings) aligns repository settings with the configurations defined in the [.github/settings.yml](https://github.com/cncf/glossary/blob/main/.github/settings.yml) file.
- To update permissions accurately, we must modify the [cncf/people](https://github.com/cncf/people/blob/main/config.yaml#L509) repository accordingly (example: [cncf/people #143](https://github.com/cncf/people/pull/143)).

**Comment by Imtiaz1234** [comment]
Understood the policy @seokho-son  😊

**Comment by waleed318** [comment]
Acknowledged @seokho-son

**Comment by ydFu** [comment]
@seokho-son 
I have understood this regulation and will comply with the Policies for localization approvers.

**Comment by pichuang** [comment]
@seokho-son Understood and will comply with Policies

**Comment by guillaumebernard84** [comment]
Hello @seokho-son , I have read and understood the policy 👍

**Comment by tym83** [comment]
@seokho-son
I have understood this regulation and will comply with the Policies for localization approvers.

**Comment by bishal7679** [comment]
@seokho-son
I have understood this regulation and will comply with the Policies for localization approvers :+1:

**Comment by justankiit** [comment]
I have understood all this regulations  @seokho-son, and will comply with the Policies for localization approvers 👍

**Comment by aj11anuj** [comment]
@seokho-son 
I have understood and will abide by these policies 👍

**Comment by tico88612** [comment]
@seokho-son I have understood these policies.

**Comment by symys** [comment]
Hello @seokho-son  , I have read and understood the policy :)

**Comment by seb-835** [comment]
Hello @seokho-son , I have read and understood the policy :)

**Comment by asem-hamid** [comment]
@seokho-son  Understood  these policies

**Comment by adowair** [comment]
@seokho-son I understand and will abide by these policies. Thanks!

**Comment by seifrajhi** [comment]
Hello @seokho-son, I have understood this regulation and will comply with the Policies for localization approvers.

**Comment by mboukhalfa** [comment]
@seokho-son Acknowledged

**Comment by matteobisi** [comment]
Hello @seokho-son , read and understood. Thanks

**Comment by SaraTrap** [comment]
Hi @seokho-son, I have read and understood the policies. Thanks.

**Comment by ntheanh201** [comment]
Hi @seokho-son, I have read and understood the policies. Thanks.

**Comment by thuanpham582002** [comment]
hello @seokho-son, I have read and understood the policies. Thanks!

**Comment by xehrad** [comment]
Hello @seokho-son 
I have read and understood the policy.

**Comment by mamrezb** [comment]
Hello @seokho-son 
I have read and understood the policies. Thanks.

**Comment by Moh0ps** [comment]
Hi @seokho-son 
I’ve read and understood the Localization policies and agree to follow these guidelines.

**Comment by b-beyaz** [comment]
hi @seokho-son 
i read and understood the policy
